src/Build.UnitTests/Graph/GraphLoadedFromSolution_tests.cs

```diff
@@ -194,7 +194,7 @@ public static IEnumerable<object[]> GraphsWithUniformSolutionConfigurations
             }
         }
 
-        [Theory]
+        [Theory(Timeout = 20_000)]
```

Comment:
Comment magic numbers please
```suggestion
        [Theory(Timeout = 20_000)] // Test hangs intermittently: https://github.com/dotnet/msbuild/issues/5520
```